### PR TITLE
Add operator to parkings

### PR DESCRIPTION
**Pull request by dvdmrtnz**
This PR adds the operator tag to parkings. This can be specially useful for paid parkings where you may want to know which parking app you are gonna have to use to pay before getting there.

### CODE REVIEW

File: indexer/ftypes_matcher.hpp [hunk 1]
```diff
@@ -126,6 +126,16 @@ class IsBicycleRentalChecker : public BaseChecker
...
 };
 
+class IsParkingChecker : public BaseChecker
+{
+  IsParkingChecker();
+
+public:
+  DECLARE_CHECKER_INSTANCE(IsParkingChecker);
+
+  uint32_t GetType() const;
```
Comment (vng) [inline indexer/ftypes_matcher.hpp:136]: Not needed

**Review by vng — APPROVED** [review]

File: indexer/ftypes_matcher.cpp [hunk 2]
```diff
@@ -241,6 +241,12 @@ IsBicycleRentalChecker::IsBicycleRentalChecker()
   m_types.push_back(c.GetTypeByPath({"amenity", "bicycle_rental"}));
 }
 
+IsParkingChecker::IsParkingChecker()
```
Comment (vng) [inline indexer/ftypes_matcher.cpp:244]: Add ``` : BaseChecker(2 /* level */)``` like in others

Comment (dvdmrtnz) [reply indexer/ftypes_matcher.cpp]: Done

File: indexer/ftypes_matcher.hpp (hunk 1 above)
Comment (dvdmrtnz) [reply indexer/ftypes_matcher.hpp]: Removed

**Pull request merged**